app.py

```diff
@@ -43,11 +43,13 @@ def do_admin_login():
     if request.method == "POST":
         json_dict = request.get_json()  # Creates into a dictionary
         newUser = User(json_dict)  # Dict will be parsed in constructor
-
-        if newUser.validateUser(): 
-            return jsonify({u'auth': u'success'})
+        uid = newUser.getUserUid()
+        if newUser.validateUser() is True: 
+            print("HERE")
+            return jsonify({u'userId': uid})
         else: 
-            return jsonify({u'auth': u'failure'})
+            print("ERROR")
+            return jsonify({u'auth': u'-1'})
```

Comment:
Not sure, but I think we should always send back the same key for the key value pair. If I try to do getString("userId") but an "auth" is sent back we'll get an exception error.

Reply:
Fixed!